instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -133,17 +133,22 @@ public boolean getBooleanProperty(String name, boolean defaultValue) {
 
   public boolean isInstrumentationEnabled(
       Iterable<String> instrumentationNames, boolean defaultEnabled) {
+    return getInstrumentationBooleanProperty(instrumentationNames, "enabled", defaultEnabled);
+  }
+
+  public boolean getInstrumentationBooleanProperty(
```

Comment:
Either this name or method's content is misleading. Name says about any boolean property. Content still talks about enablement and contains quite specific logic.

Reply:
True - since a boolean property is always about enabling/disabling something, I change it to `isInstrumentationPropertyEnabled`